docker/docker-compose.yml

```diff
@@ -1,105 +1,266 @@
-version: '3.7'
+# TODO: Store logs from app on local machine
```

Comment:
is this done?

Reply:
No, do we need it?

Reply:
it will could make debugging/troubleshooting easier

Reply:
That's done. Logs will be stored in web-local/target/logs.